test/app/app.rb

```diff
@@ -24,4 +24,4 @@
 org = Organization.create!(name: 'test')
 org.users.create!(email: 'squirrel@example.com')
 
-User.find_by(email: 'squirrel@example.com')
+User.find_by(email: 'squirrel@example.com') # bump
```

Comment:
Fuzzed Data: Shiba doesn't know the size of **users**.  For these purposes we set the table size to **5000**.
Table Scan: The database reads **100%** (5000) of the rows in **users**, skipping any indexes.
Small Results: The database returns 1 row(s) to the client.
 (1550729070